### PR TITLE
Test docker image during CI

### LINKED ISSUE #110: Test Docker image during CI
It would be nice to test the Docker image and verify, that it still builds.

### PR DESCRIPTION
**Pull request by dbrgn**
WIP.

Fixes #110.

### CODE REVIEW

**Comment by dbrgn** [comment]
Hm, with this we're basically testing the SpaceAPI twice: Once in Travis and once inside the container...

Any ideas, @rnestler?

We could of course skip the testing in Travis and only run the docker build. We could then exec into the container and run the tests in there.

**Comment by dbrgn** [comment]
> We could then exec into the container and run the tests in there.

Oh, looks like that won't work, since we delete the source after building :(

**Comment by rnestler** [comment]
As a basic test I think it's enough to show that the container still builds. The build may run in travis but fail in Docker.

File: .travis.yml [hunk 1]
```diff
@@ -1,6 +1,13 @@
-language: rust
-sudo: false
-rust:
-  - stable
+sudo: required
```
Comment (rnestler) [inline .travis.yml]: Why do we require sudo? Because otherwise it would be docker in docker?

**Pull request closed without merging**